package.json

```diff
@@ -4,18 +4,21 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.3.15",
+    "@material-ui/core": "^4.11.3",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "apollo-server": "^2.23.0",
     "apollo-server-express": "^2.23.0",
     "express": "^4.17.1",
+    "formik": "^2.2.6",
     "graphql": "^15.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
+    "web-vitals": "^1.0.1",
+    "yup": "^0.32.9"
```

Comment:
This is real nit-picky but after we get the entire project deployed and are comfortable with that process we should remove all `^` to packages in the our dependencies list. The reason being to avoid any unnecessary headaches in the future with dependency issues/bug versions that get released on accident.

Reply:
We can definitely come back to this, assuming this project has a lifespan that makes this a valid concern.